core/client/fs/src/main/java/alluxio/client/file/options/OpenFileOptions.java

```diff
@@ -246,23 +265,26 @@ public boolean equals(Object o) {
     }
     OpenFileOptions that = (OpenFileOptions) o;
     return Objects.equal(mCacheLocationPolicy, that.mCacheLocationPolicy)
+        && Objects.equal(mPartialCache, that.mPartialCache)
         && Objects.equal(mReadType, that.mReadType)
         && Objects.equal(mMaxUfsReadConcurrency, that.mMaxUfsReadConcurrency)
         && Objects.equal(mUfsReadLocationPolicy, that.mUfsReadLocationPolicy);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hashCode(mCacheLocationPolicy, mReadType, mMaxUfsReadConcurrency);
+    return Objects.hashCode(mCacheLocationPolicy, mPartialCache, mReadType,
+        mMaxUfsReadConcurrency, mUfsReadLocationPolicy);
   }
 
   @Override
   public String toString() {
     return Objects.toStringHelper(this)
-        .add("cacheLocationPolicy", mCacheLocationPolicy)
-        .add("maxUfsReadConcurrency", mMaxUfsReadConcurrency)
-        .add("readType", mReadType)
-        .add("ufsReadLocationPolicy", mUfsReadLocationPolicy)
+        .add("Cache Location Policy", mCacheLocationPolicy)
```

Comment:
Q: is this the new convention? it seems consistent with previous code style in the code base. I slightly incline to the previous convention

Reply:
Updated, thanks.